docs/source/en/conceptual_guides/intro_agents.mdx

```diff
@@ -77,7 +77,7 @@ See? With these two examples, we already found the need for a few items to help
 - Of course, an LLM that acts as the engine powering the system
 - A list of tools that the agent can access
 - A parser that extracts tool calls from the LLM output
-- A system prompt synced with the parser
+- A system prompt indicating tool calling formats that match the format expected by the action parser
```

Comment:
Maybe the system prompt should be mentioned before the need of an aligned action parser.
Nit: maybe a rewording so the word "format" does not appear twice in the same sentence...